test/powershell/Language/Scripting/ScriptHelp.Tests.ps1

```diff
@@ -614,6 +614,20 @@ Describe 'get-help other tests' -Tags "CI" {
         It '$x.Parameters.parameter[2].defaultValue' { $x.Parameters.parameter[2].defaultValue | Should -BeExactly 'parameter is mandatory' }
     }
 
+    Context 'get-help helpFunc14' {
+        function helpFunc14
+        {
+            param(
+                [SupportsWildcards()]
+                $p1
```

Comment:
can you add a property which does not have `SupportsWildcards` attribute and validate the value of globbing.

Reply:
Added helpFunc15 to test globbing is set to false when no `[SupportsWildcard]` attribute is present.